qiskit_sphinx_theme/__init__.py

```diff
@@ -6,14 +6,108 @@
+QISKIT_SLACK = 'https://qiskit.slack.com'
+QISKIT_TEXTBOOK = 'https://qiskit.org/textbook-beta'
+QISKIT_EVENTS = 'https://qiskit.org/events'
+QISKIT_PARTNERS_DOCS = 'https://qiskit.org/documentation/partners'
+QISKIT_EXPERIMENTS_DOCS = 'https://qiskit.org/documentation/experiments'
+QISKIT_ORGANIZATION_GITHUB = 'https://github.com/Qiskit/'
+
+QISKIT_ORG_LINK = {
+    'label': 'Qiskit home page',
+    'url': QISKIT_ORG
+}
+
+TOP_MENU_LINKS = {
+    'qiskit_core_getting_started': {
+        'label': 'Getting Started',
+        'path': 'getting_started'
+    },
+    'qiskit_core_tutorials': {
+        'label': 'Tutorials',
+        'path': 'tutorials'
+    },
+    'qiskit_partners': {
+        'label': 'Partners',
+        'url': QISKIT_PARTNERS_DOCS,
+    },
+    'qiskit_applications': {
+        'label': 'Applications',
+        'children': [
+            {
+                'label': 'Machine learning',
+                'url': QISKIT_MACHINE_LEARNING_DOCS,
+                'description': 'QSVM, VQC (Variational Quantum Classifier), and QGAN (Quantum Generative Adversarial Network) algorithms.'
+            },
+            {
+                'label': 'Nature',
+                'url': QISKIT_NATURE_DOCS,
+                'description': 'Quantum applications in chemistry, physics, and biology.'
+            },
+            {
+                'label': 'Finance',
+                'url': QISKIT_FINANCE_DOCS,
+                'description': 'Uncertainty components for stock/securities problems, Ising translators for portfolio optimizations and data providers to source real or random data.'
+            },
+            {
+                'label': 'Optimization',
+                'url': QISKIT_OPTIMIZATION_DOCS,
+                'description': 'High-level optimization problems that are ready to run on simulators and real quantum devices'
+            }
+        ]
+    },
+    'qiskit_experiments': {
+        'label': 'Experiments',
+        'url': QISKIT_EXPERIMENTS_DOCS,
+    },
+    'resources': {
+        'label': 'Resources',
+        'children': [
+            {
+                'label': 'Slack support',
+                'url': QISKIT_SLACK
+            },
+            {
+                'label': 'Qiskit Textbook',
+                'url': QISKIT_TEXTBOOK
+            },
+            {
+                'label': 'Qiskit events',
+                'url': QISKIT_EVENTS
+            }
+        ]
+    },
+    'qiskit_organization_github': {
+        'label': 'GitHub',
+        'url': QISKIT_ORGANIZATION_GITHUB
+    }
+}
 
 def get_html_theme_path():
     """Return list of HTML theme paths."""
     cur_dir = path.abspath(path.dirname(path.dirname(__file__)))
     return cur_dir
 
-# See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
+def map_to_theme_links(app, config):
+    """Map the theme options given in the config file to links"""
+    theme_options = config.html_theme_options or {}
```

Comment:
Does `config.html_theme_options` equal `None` if the `conf.py` file doesn't set it, or will it be an attribute error?

Reply:
I don't know, tbh 🤔 . I'm gonna try and see what happen . Thank you! I didn't though on that!